frontend/src/components/toolbar/problem-metadata/problem-metadata.component.tsx

```diff
@@ -55,18 +59,39 @@ export function ProblemMetadataComponent() {
   //#region handlers
   const handleSubmit = async (event) => {
```

Comment:
Myślę, że jeszcze lepiej będzie jeżeli to sprawdzenie będzie odbywać się nie pod czas submitu, a pod czas kiedy będzie utracony focus z tego pola tekstowego. 
Czyli w jakiś sposób wykorzystać `onBlur`
https://stackoverflow.com/questions/37609049/how-to-correctly-catch-change-focusout-event-on-text-input-in-react-js